index.js

```diff
@@ -27,6 +27,7 @@ function Hoodie (options) {
 
   api.account = new Account({ url: api.url + '/account/api' })
   api.task = new Task('/hoodie/task/api')
+  api.request = require('./lib/request')(api.url)
```

Comment:
Is this calling the `request` function with the `api.url`? I'm not sure I understand what this is doing.